### PR TITLE
Conditional Statements and Data Types

### PR DESCRIPTION
**Pull request by GarinaKristina**

### CODE REVIEW

File: 3.1str+boolean.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let a = "apple" ;
```
Comment (YauhenEp) [inline 3.1str+boolean.js:1]: use const when variable will not change

File: 3.1str+boolean.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+let b = true;
```
Comment (YauhenEp) [inline 3.1str+boolean.js:2]: const

File: 3.1str+boolean.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+let c = (2+1);
```
Comment (YauhenEp) [inline 3.1str+boolean.js:3]: const

**Comment by GarinaKristina** [comment]
work on bugs

**Pull request merged**